### PR TITLE
Fixing no title on new bookmark

### PR DESCRIPTION
**Pull request by andrewconner**
@raymondng @stkem

### CODE REVIEW

**Comment by stkem** [comment]
lgtm

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepInterner.scala [hunk 1]
```diff
@@ -209,7 +211,7 @@ class KeepInterner @Inject() (
         }
 
         val savedKeep = bookmark.copy(
-          title = title orElse bookmark.title orElse uri.title,
+          title = trimmedTitle orElse bookmark.title orElse uri.title,
```
Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepInterner.scala:214]: Not sure how pervasive the issue is -- do we need to do the same check for `bookmark.title` and `uri.title`?

Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepInterner.scala:214]: Not an issue I've noticed, just did a little cleanup.

Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepInterner.scala:214]: Oh cool

**Pull request merged**